pkg/webview/view.proto

```diff
@@ -4,8 +4,79 @@ package webview;
 
 import "google/api/annotations.proto";
 
-// TODO(dmiller) fill this in
-message Resource {}
+enum TriggerMode {
+  TRIGGER_MODE_AUTO = 0;
+  TRIGGER_MODE_MANUAL = 1;
+}
+
+message BuildRecord {
+  repeated string edits = 1;
+  string error = 2;
+  repeated string warnings = 3;
+  string start_time = 4;
```

Comment:
I might add a note that this is only a string for backwards compatibility. Future time fields should use the built in proto type